Assets/RedBlueGames/BulkRename/Editor/BulkRenamerWindow.cs

```diff
@@ -211,64 +210,33 @@ private void OnGUI()
             EditorGUILayout.EndScrollView();
         }
 
-        private string[] GetNamesFromSelections()
+        private void SortObjectsToRename()
```

Comment:
I consolidated the sorting into this one function that runs on the list of selected objects. This means `GetNamesFromSelection` and `RenameAssets` could continue to operate on `this.objectsToRename`.